Loop/View Controllers/SettingsTableViewController.swift

```diff
@@ -429,17 +429,17 @@ final class SettingsTableViewController: UITableViewController {
                 present(alert, animated: true, completion: nil)
             } else {
                 // Add new CGM
-                let cgmManagers = allCGMManagers.compactMap({ $0 as? CGMManagerUI.Type })
+                let cgmManagers = dataManager.availableCGMManagers
 
                 switch cgmManagers.count {
                 case 1:
-                    if let CGMManagerType = cgmManagers.first {
+                    if let cgmManager = cgmManagers.first, let CGMManagerType = dataManager.cgmManagerTypeByIdentifier(cgmManager.identifier) {
```

Comment:
[nit] Capitalization of variable?

Reply:
In the end it is used for initialization. So it is a class like. Again, just trying to follow existing patterns.